src/js/eva-config.js

```diff
@@ -114,154 +104,541 @@ consequenceTypesColors = [
     {id: 'intergenic_variant', color: '#636363', impact: 'MODIFIER'}
 ];
 
-consequenceTypes = [
-    {
-        name: 'Transcript Variant',
-        cls: "parent",
-        expanded: true,
-        leaf: false,
-        checked: false,
-        iconCls: 'no-icon',
-        children: [
-            {
-                name: 'Coding Variant',
+consequenceTypes = {
+    default:[
+        {acc: 'SO:0001628', name: 'intergenic_variant', qtip: 'A sequence variant located in the intergenic region, between genes', leaf: true, checked: false, iconCls: 'no-icon', color: '#636363' }
+    ],
+    78 : [
+                {
+                name: 'Transcript Variant',
```

Comment:
since its multi dimensional array I'm not sure how we can do this like which consequence types goes under which parent.
All other changes are made.